### PR TITLE
Add Example: NextJs-Together-Lllamaindex

### PR DESCRIPTION
**Pull request by martinibach**
To Dos:

- [x] Adapt Readme, with step-by-step tutorial
- [ ] Add to restack docs with step-by-step tutorial

### CODE REVIEW

File: examples/nextjs-together-llamaindex/frontend/src/app/components/Examples.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const Examples = () => {
+  const [loading, setLoading] = useState(false);
+  const [output, setOutput] = useState({});
+
+  const handleButtonClick = async (example: Example) => {
+    setLoading(true);
+    setOutput({});
+    try {
+      const result = await triggerWorkflow(example.workflowName, example.input);
```
Comment (omaiboroda) [inline examples/nextjs-together-llamaindex/frontend/src/app/components/Examples.tsx:15]: Hello, it's Nomenclature Police👮‍♀️
You have violated RNO p12 3a where workflows are _scheduled_, not _triggered_.
Please comply to protocol else you can be arrested.

On duty, your Nomenclature Police 🚓

File: examples/nextjs-together-llamaindex/backend/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  },
+  "nodemonConfig": {
+    "execMap": {
+      "ts": "ts-node"
+    },
+    "ext": "ts",
+    "watch": [
+      "src"
+    ]
+  },
```
Comment (omaiboroda) [inline examples/nextjs-together-llamaindex/backend/package.json:21]:
```suggestion
  "nodemonConfig": {
    "execMap": {
      "ts": "ts-node"
    },
    "ext": "ts",
    "watch": [
      "src"
    ]
  },
```
not used?

File: examples/nextjs-together-llamaindex/backend/src/services.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                    // add other functions here
+                },
+            }),
+            // Start the together service to queue function calls to the Together API with rate limiting
+            // https://docs.together.ai/docs/rate-limits
+            client.startService({
+                taskQueue: 'together',
+                functions: { togetherChatCompletionBasic, llamaIndexQueryTogether },
+                options: {
+                    rateLimit: (60 / 60), // 60 RPM -> 1 RPS
```
Comment (omaiboroda) [inline examples/nextjs-together-llamaindex/backend/src/services.ts:26]: 🖤

File: examples/nextjs-together-llamaindex/README.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+node restack_up.mjs
+```
+
+For detailed deployment information, see the [Restack Cloud documentation](https://docs.restack.io/restack-cloud/deployrepo).
+
+## Project Components
+
+### Frontend
+
+- Modern Next.js 14 application
```
Comment (omaiboroda) [inline examples/nextjs-together-llamaindex/README.md:126]: 👴 latest next is 15

**Review by omaiboroda — APPROVED** [review]

File: examples/nextjs-together-llamaindex/frontend/src/app/components/Examples.tsx (hunk 1 above)
Comment (martinibach) [reply examples/nextjs-together-llamaindex/frontend/src/app/components/Examples.tsx:15]: Yes this is from existing NextJs example, will fix.

**Pull request merged**